### ISSUE TITLE
V2.2.3 - horizontal button stack not navigating when clicked

### ISSUE THREAD

**Issue by SneakieGargamel** [Clooos/Bubble-Card#733]
**Describe the bug**  
When I click one of the buttons in a horizontal button stack to navigate to another page, nothing happens. I use it as a footer navigation bar.

**To Reproduce** 
Press one of the buttons in a horizontal button stack when used for navigation

**Expected behavior**  
When I click one of the buttons it should navigate to the specific view.

**Screenshots**  
![IMG_3731](https://github.com/user-attachments/assets/6be56728-ad73-42f1-a137-28b96e865d39)

**YAML**  
If applicable, add any relevant YAML code.

```yaml
type: custom:bubble-card
card_type: horizontal-buttons-stack
is_sidebar_hidden: false
highlight_current_view: false
rise_animation: true
hide_gradient: false
1_link: /lovelace
1_icon: mdi:home
2_link: /lovelace/energy
2_icon: mdi:lightning-bolt
3_link: /lovelace/lights
3_icon: mdi:lightbulb
4_link: /lovelace/cctv
4_icon: mdi:cctv
5_link: /lovelace/admin
5_icon: mdi:shield-crown-outline
styles: |
  .lovelace > .icon {
    color: var(--primary-color) !important;
  }
  .lovelace\\/energy > .icon {
    color: var(--yellow-color) !important;
  }
  .lovelace\\/lights > .icon {
    color: var(--amber-color) !important;
  }
  .lovelace\\/cctv > .icon {
    color: var(--purple-color) !important;
  } 
```

**Informations (please complete the following information):**  
 - OS: iOS
 - Browser/App: App
 - Bubble Card version: 2.2.3
 - Home Assistant version: 2024.8.2

Thank you! 🍻

**Comment by Clooos** [comment]
Thank you for the feedback, I will check that.

**Comment by SneakieGargamel** [comment]
> Thank you for the feedback, I will check that.

Thank you!

**Comment by benswinney** [comment]
Experiencing similar issue.

I recreated the navigation bar and I still have similar problems.

One thing I noticed when I checked using the UI on a web browser is that instead of navigating to the correct page when the button is pressed, it instead appends the link to the original with a # at the start.

Before clicking the Navigation button

```
https://<ha-instance>:8123/lovelace/0
```

After clicking the navigation button
```
https://<ha-instance>:8123/lovelace/0#/lovelace/1
```

Hope this helps with debugging.
